SimPEG/EM/NSEM/FieldsNSEM.py

```diff
@@ -0,0 +1,563 @@
+from SimPEG import Survey, Utils, Problem, np, sp, mkvc
+from scipy.constants import mu_0
+import numpy as np
```

Comment:
numpy is imported twice here from SimPEG and from numpy. @grosenkj could you remove one?